### ISSUE TITLE
Circular perimeter and core solve adjacencies 

### ISSUE THREAD

**Issue by antonszilasi** [mostaphaRoudsari/Honeybee#583]
@chriswmackey  @mostaphaRoudsari 

Gentlemen yesterday I tried to make a circular energy model where the perimeter zone is circular and the core is circular. See below:

![perimeter and core circular](https://cloud.githubusercontent.com/assets/8773483/21230233/e874e996-c2a9-11e6-8381-738a2aa9e37a.jpg)

I found that whatever tolerance I set in the input of the solve adjacency component, the component would set all the surfaces to exposed to outdoors in other words it wasn't picking up adjacencies:

![solve adjacencies](https://cloud.githubusercontent.com/assets/8773483/21230363/57559c0c-c2aa-11e6-9629-e76dbe78e67b.JPG)

I only got it to work correctly by commenting out the code here:
![commented out code](https://cloud.githubusercontent.com/assets/8773483/21230471/bb1e8f3c-c2aa-11e6-848e-bbb1c87833f3.JPG)

Now it works correctly see:

![adjacencies working correctly](https://cloud.githubusercontent.com/assets/8773483/21230569/07164ba0-c2ab-11e6-9200-4400790ee971.JPG)

In other words it wasn't picking up those adjacencies because the centroid of each zone was within the absolute model tolerance.

Should we have an input on the component that can change the tolerance is this part of the code? This could help users that can't code

**Comment by mostaphaRoudsari** [comment]
It doesn't have to do with tolerance. It's happening because the center of the core and perimeter are the same. You need to break down the perimeter zone into pieces (North, East, West, South) and then it will work. This is a very unique, and from energy modeling point of view wrong, form of perimeter and core zone.

**Comment by chriswmackey** [comment]
I agree with @mostaphaRoudsari that this is not the correct way to zone a space like this.  Also, your edit @antonszilasi , is not fixing the issue as the zone is essentially making the zones adjacent to themselves.

-Chris
